control/iosys.py

```diff
@@ -108,6 +108,14 @@ class for a set of subclasses that are used to implement specific
       The default is to return the entire system state.
 
     """
+
+    idCounter = 0
```

Comment:
Make this an instance variable?

Reply:
This is the class variable used to count how many object instances were created so far (new instances without names are called sys[idCounter]). Is there a naming convention I am not following (eg. should class variables be prefixed by an underscore (_idCounter))?

Reply:
Ok, my bad. Sorry.